pramen/core/src/main/scala/za/co/absa/pramen/core/reader/TableReaderJdbcNative.scala

```diff
@@ -35,13 +35,14 @@ class TableReaderJdbcNative(jdbcReaderConfig: TableReaderJdbcConfig,
   private val log = LoggerFactory.getLogger(this.getClass)
 
   private val jdbcConfig = jdbcReaderConfig.jdbcConfig
-  private val infoDateFormatPattern = jdbcReaderConfig.infoDateFormat
-  private val infoDateFormatter = DateTimeFormatter.ofPattern(infoDateFormatPattern)
+
   private val url = jdbcUrlSelector.getWorkingUrl(jdbcConfig.retries.getOrElse(jdbcUrlSelector.getNumberOfUrls))
 
   logConfiguration()
 
-  private[core] def getJdbcReaderConfig: TableReaderJdbcConfig = jdbcReaderConfig
+  private[core] def getJdbcReaderConfig: TableReaderJdbcConfig = {
+    jdbcReaderConfig.copy(jdbcConfig = jdbcConfig)
```

Comment:
I'm just curious, why do you need to do this instead of just `jdbcReaderConfig.copy()`?

Reply:
This is to update `jdbcConfig` with `fetchsize` option. Currently, `fetchsize` is set on the reader level of configuration, and I want to propagate it to `jdbcConfig` for backwards compatibility.